pkg/cmd/server/start/command_test.go

```diff
@@ -170,9 +171,9 @@ func TestCommandBindingNodesForMaster(t *testing.T) {
 	actualCfg := executeMasterCommand([]string{"master", "--nodes=" + valueToSet})
 
 	expectedArgs := NewDefaultMasterArgs()
-	expectedArgs.NodeList.Set(valueToSet)
+	expectedArgs.NodeList = append(expectedArgs.NodeList, strings.Split(valueToSet, ",")...)
 
-	if expectedArgs.NodeList.String() != actualCfg.NodeList.String() {
+	if fmt.Sprint(expectedArgs.NodeList) != fmt.Sprint(actualCfg.NodeList) {
```

Comment:
I'd ask who wrote it, but I'm afraid of the answer.  Since you're here, can you fix it?